### PR TITLE
add makefile

### PR DESCRIPTION
**Pull request by mouuii**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.PHONY: build
+build:
+	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
```
Comment (kilosonc) [inline Makefile:3]: Could you add more about building job and swagger, cleaning and making images?

**Review by kilosonc — CHANGES_REQUESTED** [review]
Please see comments below

File: Makefile (hunk 1 above)
Comment (mouuii) [reply Makefile:3]: added

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	docker build -t horizon -f build/job/Dockerfile . --platform linux/arm64
+else
+	docker build -t horizon -f build/job/Dockerfile .
+endif
+
+
+.PHONY: clean
+clean:
+	docker rmi -f horizon-swagger
+	docker rmi -f horizon
```
Comment (kilosonc) [inline Makefile:30]: empty line

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.PHONY: build
+build:
+	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
+
+.PHONY: swagger-build
+swagger-build:
```
Comment (kilosonc) [inline Makefile:6]: ‘swagger’ perhaps? 'swagger' below should be 'swagger-run', keep up with ‘horizon’ command.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ifeq ($(shell uname -m),arm64)
+	docker build -t horizon-swagger -f build/swagger/Dockerfile . --platform linux/arm64
+else
+	docker build -t horizon-swagger -f build/swagger/Dockerfile .
+endif
+
```
Comment (kilosonc) [inline Makefile:12]: Should add command for building job and cleaning.

**Review by kilosonc — CHANGES_REQUESTED** [review]
Please consider comments below

File: Makefile (hunk 3 above)
Comment (mouuii) [reply Makefile]: changed

File: Makefile (hunk 2 above)
Comment (mouuii) [reply Makefile]: added

File: Makefile (hunk 4 above)
Comment (mouuii) [reply Makefile:12]: added

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+.PHONY: swagger-run
+swagger-run: swagger
+	@echo "http://localhost:80"
+	docker run -p 80:8080 horizon-swagger
```
Comment (kilosonc) [inline Makefile:16]: docker run --rm?

Comment (mouuii) [reply Makefile]: added

**Review by xuzhu-591 — APPROVED** [review]

**Review by iamyeka — APPROVED** [review]

**Review by kilosonc — APPROVED** [review]
LGTM

**Pull request merged**